pbjam/session.py

```diff
@@ -532,68 +544,92 @@ class session():
     def __init__(self, ID=None, numax=None, dnu=None, teff=None, bp_rp=None,
                  timeseries=None, spectrum=None, dictlike=None, use_cached=False, 
                  cadence=None, campaign=None, sector=None, month=None, 
-                 quarter=None, mission=None, path=None, download_dir=None):
-
-        self.stars = []
-        self.references = references()
-        self.references._addRef(['python', 'pandas', 'numpy', 'astropy', 
-                                 'lightkurve'])
+                 quarter=None, mission=None, path=None, download_dir=None, 
+                 session_ID=None, level='DEBUG'):
```

Comment:
'level' as an variable name is ambiguous I think.

Can we replace this with verbosity, loggingLevel or something along those lines?

Reply:
I'm happy to change the var name to `logging_level`, since verbosity could be confused with console output, but this setting only determines the level of the file logging handler.

As an aside: If a user did want to change the console logging level (e.g. to 'DEBUG' level) they could just do:

```python
from pbjam import console_handler

console_handler.setLevel('DEBUG')
```

Reply:
The var name can be more appropriate, up to you. 

'level' is just ambiguous is all.